Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocBooleanValueImplementation/Rendering/BocBooleanValueRendererTest.cs

```diff
@@ -141,23 +141,23 @@ public void RenderTrue ()
     {
       _booleanValue.Setup (mock => mock.Enabled).Returns (true);
       _booleanValue.Object.Value = true;
-      CheckRendering ("true", true.ToString(), "TrueIconUrl", _booleanValue.Object.TrueDescription);
+      CheckRendering ("true", true.ToString(), "TrueIconUrl", _booleanValue.Object.TrueDescription.ToString (WebStringEncoding.HtmlWithTransformedLineBreaks));
```

Comment:
Given the uniformity of this change, I would recommend changing the parameter type and passing webstring instead of string to improve maintainability: 
1. if we need to change/add tests, they also require the "long" form.
2. if we change this now instead of later, we only have a single place in the code where this change happens instead of changing 10 occurances now and then changing the same 10 occurances later on, reducing effort down the line and the total affectd changeset.

against this speaks the fact that value isn't passed as a nullable boolean but also a string. That was done to have more control over the passed string in case a test needs to do something special. I'm not sure I would still do it this way to today instead of passing a nullable boolean.

pro/cons for this refactoring?

Reply:
Changing it makes sense